_includes/navbar.html

```diff
@@ -98,36 +98,46 @@
               </li>
             </ul>
           </li>
-          <li id="guides-nav-item">
+          <li>
```

Comment:
Was this style not doing anything?

Reply:
Oh - I think I see - it did have custom styling - but is no longer necessary. Please feel free to disregard!